pkg/actors/actors.go

```diff
@@ -762,90 +753,79 @@ func (a *actorsRuntime) startReminder(reminder *Reminder, stopChannel chan bool)
 			break
 		}
 
-		err = a.executeReminder(reminder.ActorType, reminder.ActorID, reminder.DueTime, reminder.Period, reminder.Name, repetitions, reminder.Data)
-		if err != nil {
-			log.Errorf("error executing reminder: %s", err)
-		}
-
-		log.Debugf("repetitions left for actors with id %s are %d", reminder.ActorID, repetitions.getValue())
-
-		if reminder.Period != "" {
+		for {
+			// TODO(dmitsh) clarify why we need the block below
```

Comment:
What is your question here? We should answer it in this PR and not leave the TODO :)

Reply:
yes, that's the goal to resolve all TODOs before merging :)